modules/oxide/src/less/theme/content/comments/comments.less

```diff
@@ -2,13 +2,46 @@
 // Comments
 //
 
-@document-comment-background-color: #fff0b7;
-@document-comment-active-background-color: #ffe168;
+@document-comment-color: #ffe89d;
+@document-comment-active-color: #ffe168;
+@document-comment-background-color: @document-comment-color;
+@document-comment-active-background-color: @document-comment-active-color;
+@document-comment-outline-color: @document-comment-color;
+@document-comment-active-outline-color: @document-comment-active-color;
 
-.tox-comments-visible .tox-comment {
-  background-color: @document-comment-background-color;
-}
+.tox-comments-visible {
+  .tox-comment[contenteditable="false"],
+  span.tox-comment img,
+  span.tox-comment > audio,
+  span.tox-comment > video,
+  span.tox-comment span.mce-preview-object {
+    &:not([data-mce-selected]) {
+      outline: 3px solid @document-comment-outline-color;
+    }
+  }
+
+  .tox-comment[contenteditable="false"][data-mce-annotation-active="true"] {
+    &:not([data-mce-selected]) {
+      outline: 3px solid @document-comment-active-outline-color;
+    }
+  }
+
+  span.tox-comment[data-mce-annotation-active="true"] {
+    & img,
+    & > audio,
+    & > video {
```

Comment:
Does this also need `span.mce-preview-object` to be included in this list?

Reply:
Yeah, it should be. Thanks for that. I am working on adding a test to cover that

Reply:
Done